### PR TITLE
Update cli args documentation

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Updating the cli documentation after 0.5 release.

### CODE REVIEW

File: docs/reference/commandline/tye-run.md [hunk 1]
```diff
@@ -43,28 +43,25 @@ If a directory path is specified, `tye run` will default to using these files, i
...
-    
-    * `debug` - display all logs that the process outputs
-    * `info` - display only informational logs
-    * `quiet` - display only warnings and errors
+    The target framework hint to use for all cross-targeting projects with multiple TFMs. This value must be a valid target framework for each individual cross-targeting project. Non-crosstargeting projects will ignore this hint and the value TFM configured in tye.yaml will override this hint.
 
-    The default value is `info`
+- `-v|--verbosity <Debug|Info|Quiet>`
+
+    The verbosity of logs emitted by `tye run`. Defaults to Info.
```
Comment (jkotalik) [inline docs/reference/commandline/tye-run.md:60]: Why did you remove the values part?

Comment (JunTaoLuo) [reply docs/reference/commandline/tye-run.md:60]: I'll revert it now I think about it. I originally made it to be more consistent with the other commands but I think the extra info is good.

**Review by jkotalik — APPROVED** [review]

**Comment by JunTaoLuo** [comment]
I was originally just going to fixup -f vs --force and --framework. But I ended up just updating all the args for tye commands since I noticed quite a few are out of sync with what we have.
